filtered-resource-service/src/main/java/uk/gov/gchq/palisade/service/filteredresource/model/AuditErrorMessage.java

```diff
@@ -142,7 +142,7 @@ default IAttributes withContext(Context context) {
             /**
              * Adds the attributes for the message.
              *
-             * @param attributes additional attributes for the request.
+             * @param attributes timestamp for the request.
```

Comment:
Is it just a timestamp that's being added to the attributes? Should this be more generalised as we don't know what information will be added in the future.